chrome/locale/en-US/zotero/zotero.properties

```diff
@@ -898,6 +898,7 @@ proxies.notification.associated.label	= Zotero automatically associated this sit
 proxies.notification.redirected.label	= Zotero automatically redirected your request to %1$S through the proxy at %2$S.
 proxies.notification.enable.button		= Enable…
 proxies.notification.settings.button	= Proxy Settings…
+proxies.notification.hide.button      = Don't Show Again
```

Comment:
Hmm, that will make https://github.com/zotero/zotero/pull/673/files#diff-72b2d297070644abfa5ac6c878f83f0dR434 less pretty, but it makes sense. Do you mind if I call it `general.dontShowAgain.button` or should we just give up all pretense of hiding the resource string values?